src/Mvc/test/Mvc.FunctionalTests/ComponentRenderingFunctionalTests.cs

```diff
@@ -40,23 +40,7 @@ public async Task Renders_BasicComponent()
             await response.AssertStatusCodeAsync(HttpStatusCode.OK);
             var content = await response.Content.ReadAsStringAsync();
 
-            AssertComponent("\n<p>Hello world!</p>", "Greetings", content);
-        }
-
-        [Fact]
-        public async Task Renders_BasicComponent_UsingRazorComponents_Prerenderer()
```

Comment:
These tests got streamlined. I did my best to remove fragility to due to extra/missing newlines.

Also there are far fewer tests because there are fewer cases to test.